WeatherApp/WeatherApp/CityScreen/CityScreenViewModel.swift

```diff
@@ -3,25 +3,25 @@ import SwiftUI
 class CityScreenViewModel: ObservableObject {
 
     private let router: RouterProtocol
-    private let weatherService: WeatherServiceProtocol
+    private let getWeatherUseCase: GetWeatherUseCaseProtocol
 
     @Published var city: String
-    @Published var weather: CurrentWeatherResponse?
+    @Published var weather: WeatherModel?
 
-    init(router: RouterProtocol, service: WeatherServiceProtocol, city: String) {
+    init(router: RouterProtocol, useCase: GetWeatherUseCaseProtocol, city: String) {
```

Comment:
i would still like this to be names `getWeatherUseCase` to make it clearer what use case is in use